pyvcloud/vcd/client.py

```diff
@@ -574,9 +575,18 @@ def set_credentials(self, creds):
         self._session.headers['x-vcloud-authorization'] = \
             response.headers['x-vcloud-authorization']
 
+        if session.get('org') == 'System':
+            self._is_loggedin_user_sys_admin = True
```

Comment:
self._is_loggedin_user_sys_admin = True if (session.get('org') == System) else False

Reply:
we don't need to set the variable to False, as it is already assigned as False as default. The If statement is more readable in this case.

Reply:
I don't see a reason why we should stick to ternary operator. Infact the styling of python ternary operator has been a topic of ragging debates where people have raised it's syntax in question. In short I feel that the simple if statement is clearer to read and doesn't comprise in any way.

ref : https://stackoverflow.com/questions/394809/does-python-have-a-ternary-conditional-operator

Reply:
okay.